index.html

```diff
@@ -143,12 +143,11 @@
             </a> -->
         </section>
         <section>
-            <img id="bio-pic" src="img/hera.jpg">
+            <img src="img/hera.jpg" alt="Hera Hussain">
```

Comment:
Can you add the `id=bio-pic` back please? Have set all images with that `id` to be the same size in the css?